### ISSUE TITLE
Disabled the auto resize event listener

### ISSUE THREAD

**Issue by shuding** [shuding/cobe#6]

**Comment by adithyaakrishna** [comment]
@shuding How about if we add a argument/flag like resizeEnabled along with the default props?

**Comment by shuding** [comment]
@adithyaakrishna Yeah sounds good, but we still need to add that to Phenomenon. Maybe we can send a PR there.
